src/data/faq.json

```diff
@@ -101,6 +101,154 @@
+                        "textblock": [
+                            "<ul>",
+                            "<li><a href='#vac_cert_multiple_scan_possible' target='_self' rel='noopener noreferrer'>Can I save my vaccination certificate on multiple smartphones, e.g., on my private and business phone?</a></li><li><a href='#vac_cert_multiple_scan_possible' target='_self' rel='noopener noreferrer'>How can I transfer the vaccination certificate when changing device?</a></li><li><a href='#vac_cert_multiple_scan_possible' target='_self' rel='noopener noreferrer'>What do I do if I lose my phone?</a></li><li><a href='#vac_cert_multiple_scan_possible' target='_self' rel='noopener noreferrer'>Will the vaccination certificate be included in my backup?</a></li><li><a href='#vac_cert_multiple_scan_possible' target='_self' rel='noopener noreferrer'>Can I manage the certificate in the Corona-Warn-App and the CovPass-App at the same time, or do I have to choose one of the two apps?</a></li>",
+                            "</ul>"
+                        ]
+                    },
+                    {
+                        "title": "What is the digital certificate of vaccination?",
+                        "anchor": "vac_cert_what",
+                        "active": true,
+                        "textblock": [
+                            "Starting with version 2.3 of the Corona-Warn-App (CWA), you can add your digital vaccination certificate in the CWA and thus securely and digitally prove your Corona vaccination status via QR code. The CWA indicates the complete vaccination protection 14 days after the last required vaccination. The digital vaccination certificate is an additional way to document vaccinations. Thereby you are able to digitally store information such as vaccination time and vaccine in a personalized manner on your smartphones. The use of digital certificate of vaccination in the CWA is voluntary. You can also prove your Corona vaccination in a different way."
+                        ]
+                    },
+                    {
+                        "title": "Why is there a digital proof of vaccination?",
+                        "anchor": "vac_cert_why",
+                        "active": true,
+                        "textblock": [
+                            "At the beginning of the year, the European Council decided to launch an interoperable and standardized proof of vaccination for medical purposes. Germany is now quickly implementing this European decision in order to be able to provide all citizens who wish to do so with a digital proof of vaccination. In the meantime, the European Commission has also presented a draft regulation regulating an EU-wide recognition framework for (digital) certificates for vaccinations, tests and for persons with COVID-19 recovered status in order to facilitate the free movement of travelers. The EU's green certificate forms the legal framework for the member states' solutions."

```

Comment:
The EU has not updated https://ec.europa.eu/info/live-work-travel-eu/coronavirus-response/safe-covid-19-vaccines-europeans/covid-19-digital-green-certificates_en yet and still refers to it there as "Digital Green Certificates".

The CWA Android app 2.4 in development refers to it as "COVID-19-Testzertifikat". The phrase translated into English has not been added to the app yet.

https://www.zusammengegencorona.de/impfen/aufklaerung-zum-impftermin/impfnachweis/ still contains the text (status 26.05.2021) "Das grüne Zertifikat der EU bildet den Rechtsrahmen für die Lösungen der Mitgliedsstaaten."

Changing "green" to "COVID-19" is probably the way to go, since it is more generic in any case.

Reply:
ok! Thanks .